### PR TITLE
testFolder

### PR DESCRIPTION
**Pull request by ArtemChumachenko**
new tests added

### CODE REVIEW

File: package-lock.json
Comment (teachandlearn999) [inline package-lock.json:1]: This file should not be checked in. Let's keep it in the original state.

Comment (ArtemChumachenko) [reply package-lock.json:1]: So nothing seems to change. I don’t know why he ended up in PR

Comment (teachandlearn999) [reply package-lock.json:1]: things did change. click on "Load diff" link to see. bunch of dependencies were added and the version got changed

Comment (ArtemChumachenko) [reply package-lock.json:1]: @teachandlearn999  I see. What can I do now? How to fix it? 
I can copy/paste the first version of this file and push it again, but I think there is a simpler solution

Comment (teachandlearn999) [reply package-lock.json:1]: will fix

Comment (ArtemChumachenko) [reply package-lock.json:1]: Did you delete this file on your own directly, as an owner?

**Review by teachandlearn999 — APPROVED** [review]

**Pull request merged**